pages/Article.jsx

```diff
@@ -0,0 +1,41 @@
+import { useParams } from 'react-router-dom'
+import ArticleCard from '../components/ArticleCard'
+import { useEffect, useState } from 'react'
+import { getArticle } from '../api'
+
+export default function Article () {
+    const [currentArticle, setCurrentArticle] = useState(null)
+    const [isError, setIsError] = useState(false)
+    const [isLoading, setIsLoading] = useState(true)
+
+    const  { article } = useParams()
+    useEffect(() => {
+        getArticle(article)
+        .then((res) => {
+            setCurrentArticle(res)
+            setIsError(false)
+        })
+        .catch(() => {
+            setIsError(true)
+        })
+        .finally(() => {
+            setIsLoading(false)
+        })
+    }, [])
+
+    if (isLoading) {
+        return <p>Loading...</p>
+    }
+
+    if (isError) {
+        return <p>Something went wrong</p>
+    }
+
+    return <article className="component">
+        <h2>{currentArticle.title}</h2>
+        <h3>by {currentArticle.author}</h3>
+        <img src={currentArticle.article_img_url}></img>
```

Comment:
would be good to have a little alt tag on here, though the articles don't come with one you can say something like 'stock photo image thumbnail for the article' just to show you've thought about it